GetandTake/Pages/Products/Product.cshtml

```diff
@@ -1,11 +1,12 @@
 ﻿@page
 @model GetandTake.Pages.Products.ProductModel
+@{
+    var products = await productService.GetAllAsync();
```

Comment:
Honestly, putting some business logic in a view is a really bad idea: in this case, your view becomes more overloaded, logic on page controller with its view is mixing, it becomes harder to read. Let's rollback to previous changes